### ISSUE TITLE
List workspaces in workspace switcher

### ISSUE THREAD

**Issue by nicOwlas** [labelflow/labelflow#351]
**Tasks**
Continuation of #350. **Feature not to be activated (use same env var as what was used to hide #350)**
- [ ] Workspace switch: In the breadcrumbs, list the available workspaces (local + onlines)
- [ ] Allow to switch workspace when clicking on a workspace name in the worspace switcher (redirects to `https://labelflow.ai/my-workspace`)

## Mockup
[Link to Figma](https://www.figma.com/file/4J9xyfF5v4B98MuKVO0Y8V/LabelFlow-mockups?node-id=2743%3A13129)
![image](https://user-images.githubusercontent.com/16015322/133427015-56530aa3-effd-4a94-944b-c56d6d6b1db7.png)

Blocked by #317 
Blocked by #296

**Comment by nicOwlas** [comment]
[UI for the workspace switcher - to be validated](https://www.figma.com/file/4J9xyfF5v4B98MuKVO0Y8V/LabelFlow-mockups?node-id=2743%3A13129)
![image](https://user-images.githubusercontent.com/16015322/133427062-d8de8177-be7b-4b7f-acc7-13d22211b035.png)

**Comment by crubier** [comment]
Use Chakra UI Avatar for the Icon

**Comment by etiennedupont** [comment]
@nicOwlas several questions on that issue:
- any UX available for the workspace name modal state?
- "Menu elements: Sign-out" I don't understand this point, is that a separate button in the navbar ?

**Comment by nicOwlas** [comment]
Nope good point I did not do it.

That makes me think that we need to add the possibility to create workspaces else the workflows will be weird.

All good when you sign up by yourself, you will be proposed to name your workspace.

But if you click on an invitation email, it creates your account and brings you on the workspace you've been invited to. But then we need to give the possibility to this user to create a workspace later on.

**Comment by nicOwlas** [comment]
The sign-out is in the user menu.

I'll split this issue into 3 tomorrow. Lets keep this issue for the workspace switcher only.

**Comment by nicOwlas** [comment]
Issue updated @etiennedupont. I kept only the elements relative to the workspace switcher. The task relative to the creation of a workspace is done #480.
The sign-in / sign-out part was already done in #350 .

**Comment by RaynalHugo** [comment]
I'd recommend waiting for https://github.com/labelflow/labelflow/pull/472 before doing this issue.

**Comment by crubier** [comment]
Please add the `Blocked by #xxx` in the bottom of description of issues please. (I jut did it, but I keep doing it every day on several issues, I'd like to not be the only one to do it properly)

**Comment by nicOwlas** [comment]
> Please add the `Blocked by #xxx` in the bottom of description of issues please. (I jut did it, but I keep doing it every day on several issues, I'd like to not be the only one to do it properly)

Don't worry, you are not the only one, there was already a blocked by #317 on this one. That's a point to validate together during the grooming. I agree that the second task of this issue needs #296.

**Comment by crubier** [comment]
Ok then when you do it, try to be holistic about it, try to look other issues and add them all at the same time. It's not really useful to add dependencies one by one without providing the full picture, because then we don't know if the issue is *really* ready, or if it's just that it has dependencies that were missed

**Comment by nicOwlas** [comment]
> Ok then when you do it, try to be holistic about it, try to look other issues and add them all at the same time. It's not really useful to add dependencies one by one without providing the full picture, because then we don't know if the issue is _really_ ready, or if it's just that it has dependencies that were missed

It's a miss, my goal is not to add them one by one (why would I do this?). We set the `eng-ready` tag together, everyone can raise his voice during the grooming if he notices I missed a dependency.
